src/components/TimelineNav/TimelineNav.module.css

```diff
@@ -280,7 +280,6 @@
   height: fit-content;
   padding: var(--eds-size-half) 0;
   /* Provides white background to fake a visual gap in the vertical line between nav items. */
-  background: var(--eds-theme-color-background-neutral-default);
```

Comment:
the visual diffs were generated b/c there was an embedded bg around the icons that was inconsistently transparent. Removing the bg fixes this bug
![Screen Shot 2023-01-10 at 17 59 56 ](https://user-images.githubusercontent.com/3056447/211689144-d63072e2-ddb9-4090-be7f-3fc2b221e210.png)

Reply:
Before:
![Screen Shot 2023-01-10 at 18 20 01 ](https://user-images.githubusercontent.com/3056447/211689258-9047e97d-ba0c-4e5f-bce3-14426482049b.png)